concise-mid-tag.cddl

```diff
@@ -74,22 +74,26 @@ identity-triple-record = [
   [ + verification-key-map ] ; DevID, or semantically equivalent
 ]
 
-pkix-base64-key-type = tstr
-pkix-base64-cert-type = tstr
+$rawkey-type-choice /= tagged-pkix-base64-key-type
+
+; Raw key in DER format base64-encoded.
+tagged-pkix-base64-key-type = #6.554(tstr)
+
+$authority-path-type-choice /= tagged-pkix-base64-cert-type
+
+; X.509 certificate chain in DER format base64-encoded
+tagged-pkix-base64-cert-type = #6.555(tstr)
 
 verification-key-map = {
-  ; Verification key in DER format base64-encoded.
-  ; Typically, but not necessarily a public key.
-  comid.key => pkix-base64-key-type
-  ; Optional X.509 certificate chain corresponding to the public key
-  ; in comid.key, encoded as an array of one or more base64-encoded
-  ; DER PKIX certificates.  The certificate containing the public key
-  ; in comid.key MUST be the first certificate.  This MAY be followed
-  ; by additional certificates, with each subsequent certificate
-  ; being the one used to certify the previous one.
-  ? comid.keychain => [ + pkix-base64-cert-type ]
+  raw-key-or-authority-path
 }
 
+raw-key-or-authority-path //=
```

Comment:
Can we simplify the naming? e.g. "key-group-choice"? (Or return to non-empty< ? key : $key-type-choice, ?authority-key : [ + $authority-type-choice]>

Reply:
I think lines 88 - 92 should be deleted

Reply:
@nedmsmith Agree, have done it along with some other comment tidy up!